Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -130,52 +130,46 @@ std::string NANDImporter::GetPath(const NANDFSTEntry& entry, const std::string&
 
 std::string NANDImporter::FormatDebugString(const NANDFSTEntry& entry)
 {
-  return fmt::format("{:12.12} {:#010b} {:#04x} {:#06x} {:#06x} {:#010x} {:#010x} {:#06x} {:#010x}",
-                     entry.name, entry.mode, entry.attr, entry.sub, entry.sib, entry.size,
-                     entry.uid, entry.gid, entry.x3);
+  return fmt::format("{:#010b} {:#04x} {:#06x} {:#06x} {:#010x} {:#010x} {:#06x} {:#010x}",
+                     entry.mode, entry.attr, entry.sub, entry.sib, entry.size, entry.uid, entry.gid,
+                     entry.x3);
 }
```

Comment:
Why was `entry.name` removed in this commit ("NANDImporter: Reduce recursion in `ProcessEntry`")?  It seems like useful information to include.

Reply:
I originally excluded it because it seemed redundant where it was being called. I re-added it though on second thought.
```
36:12:032 DiscIO/NANDImporter.cpp:141 I[DIO]: Entry: sys          0b11110010 0x00 0x001e 0xffff 0x00000000 0x00000000 0x0000 0x00000000 Path: /sys
36:12:032 DiscIO/NANDImporter.cpp:141 I[DIO]: Entry: space.sys    0b11110001 0x00 0x4b18 0x0066 0x00004ac4 0x00000000 0x0000 0x00000000 Path: /sys/space.sys
36:12:032 DiscIO/NANDImporter.cpp:141 I[DIO]: Entry: uid.sys      0b11110001 0x00 0x02e1 0x000e 0x0000054c 0x00000000 0x0000 0x00000000 Path: /sys/uid.sys
36:12:032 DiscIO/NANDImporter.cpp:141 I[DIO]: Entry: cert.sys     0b11110101 0x00 0x5569 0xffff 0x00000a00 0x00000000 0x0000 0x00000000 Path: /sys/cert.sys
```